src/Analysis/Engine/Test/LanguageServerTests.cs

```diff
@@ -992,6 +992,19 @@ class C:
             );
         }
 
+        [TestMethod, Priority(0)]
+        public async Task OnTypeFormatting() {
+            var s = await CreateServer();
```

Comment:
Don't forget to dispose server object at the end of the test.

Reply:
I can do this, but no other tests in this file dispose of the server either (which is why I didn't think to do so).

Reply:
Those are old tests we do need to fix at some point :-)

Reply:
I fixed it in this PR, and I can probably go do it in another PR for the rest of them (and move the tests out of this giant file).